src/components/ItemDetail/LineEntry.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { calculatedLink } from '../../services/helpers.js';
```

Comment:
Nice.

Reply:
Ah, now I see what it's returning... which is essentially JSX so I'd make an argument for code organization that anything returning JSX should live somewhere in `/components`.   I think of `services` as purely business logic, and no React stuff.   How about we create a new directory in `/components` to house helpers?